### PR TITLE
add entitlementManagement to unified RBAC

### PR DESCRIPTION
**Pull request by markwahl-msft**

### CODE REVIEW

**Comment by markwahl-msft** [comment]
@abhijeetsinha FYI work in progress, pending completion of internal review

**Comment by lramosvea** [comment]
@markwahl-msft is this work in progress yet?

**Comment by markwahl-msft** [comment]
#sign-off

File: api-reference/beta/api/unifiedroledefinition-get.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ Get the properties and relationships of a [unifiedRoleDefinition](../resources/u
 The following RBAC providers are currently supported:
 - cloud PC 
 - device management (Intune)
-- directory (Azure AD) 
+- directory (Azure AD directory roles)
+- entitlementManagement (Azure AD entitlement management)
```
Comment (Lauragra) [inline api-reference/beta/api/unifiedroledefinition-get.md:22]: Do you actually just mean to use the friendly construction "entitlement management"? This isn't s reference to a specific resource, for example.

File: api-reference/beta/resources/rbacapplication.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Role management container for unified role definitions and role assignments for Microsoft 365 RBAC providers. Currently "directory" is the only RBAC application supported.
+Role management container for unified role definitions and role assignments for Microsoft 365 RBAC providers. Currently "directory" and "entitlementManagement" are the only RBAC applications supported.
```
Comment (Lauragra) [inline api-reference/beta/resources/rbacapplication.md:16]: I don't think that the application types need to be in quotation marks. Also, should be "entitlement management"?

File: api-reference/beta/resources/rolemanagement.md [hunk 3]
```diff
@@ -42,7 +44,7 @@ None.
 |cloudPC|[rbacApplicationMultiple](rbacapplicationmultiple.md)|Provides access to role definitions and role assignments of a cloud PC RBAC provider. Read-only. Nullable.|
 |deviceManagement|[rbacApplicationMultiple](rbacapplicationmultiple.md)| Provides access to role definitions and role assignments of an Intune RBAC provider. Read-only. Nullable.|
 |directory|[rbacApplication](rbacapplication.md)|Provides access to role definitions and role assignments of an Azure AD RBAC provider. Read-only. Nullable.|
-
+|entitlementManagement|[rbacApplication](rbacapplication.md)| Provides access to role definitions and role assignments of an Azure AD entitlement management. Read-only. Nullable.|
```
Comment (Lauragra) [inline api-reference/beta/resources/rolemanagement.md:47]: There is also an entitlementManagement entity defined in the metadata that doesn't appear to be documented. Please add a topic for it.

Comment (markwahl-msft) [reply api-reference/beta/resources/rolemanagement.md:47]: Do you mean <EntityType Name="entitlementManagement"? That is not part of RBAC, it is the naming structure for collecting the entitlement management resources under /identityGovernance, it has no APIs or properties for itself.  It is documented here: https://docs.microsoft.com/graph/api/resources/entitlementmanagement-root?view=graph-rest-beta

File: api-reference/beta/api/unifiedroledefinition-get.md (hunk 1 above)
Comment (markwahl-msft) [reply api-reference/beta/api/unifiedroledefinition-get.md]: updated, thanks

File: api-reference/beta/resources/rbacapplication.md (hunk 2 above)
Comment (markwahl-msft) [reply api-reference/beta/resources/rbacapplication.md]: updated, thanks.

**Pull request merged**